libs/filamat/src/MaterialBuilder.cpp

```diff
@@ -509,9 +509,7 @@ Package MaterialBuilder::build() noexcept {
 
         // Re-populate the set of sampler bindings for this API.
         filament::SamplerBindingMap map;
-        auto backend = static_cast<filament::backend::Backend>(params.targetApi);
-        uint8_t offset = filament::getSamplerBindingsStart(backend);
-        map.populate(offset, &info.sib, mMaterialName.c_str());
+        map.populate(&info.sib, mMaterialName.c_str());
```

Comment:
I think we can move this outside of the loop now that this doesn't depend on the `targetApi`.

Reply:
Nice catch, I amended the commit.